gnu-sort/cmd/main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestSortLines(t *testing.T) {
+	tests := []struct {
+		input    []string
```

Comment:
Mostly good, just add a new field `name` that describes the purpose of a test case and then use it as a first argument for the function `t.Run`

Reply:
Should be all good now :) 
@danilo-gemoli